engine/table/src/main/java/io/deephaven/engine/table/impl/by/FloatChunkedVarOperator.java

```diff
@@ -172,15 +172,17 @@ private boolean removeChunk(FloatChunk<? extends Values> values, long destinatio
             }
             sumSource.set(destination, newSum);
             sum2Source.set(destination, newSum2);
-        } else if (totalNormalCount == 1 || forceNanResult) {
+        }  else {
+            newSum = sumSource.getUnsafe(destination);
+            newSum2 = sum2Source.getUnsafe(destination);
+        }
+
+        if (totalNormalCount == 1 || forceNanResult) {
```

Comment:
This corrects a bug where all but exactly one value are removed.  This code is now (mostly) the same as `Char` operator excluding the NaN + Inf handling.